docs/concepts/artifacts.md

```diff
@@ -178,53 +202,49 @@ if __name__ == "__main__":
     my_fn()
 ```
 
-As you can see, you don't need to create an artifact in a flow run context. You can use it however you like and still get the benefits in the Prefect UI.
+![Table artifact with customer info](/img/ui/table-artifact-info.png)
+As you can see, you don't need to create an artifact in a flow run context.
+You can use it however you like and still get the benefits in the Prefect UI.
 
-## Managing Artifacts
+## Managing artifacts
 
-### Reading Artifacts
+### Reading artifacts
 
-In the Prefect UI, you can view all of the latest versions of your artifacts and click into a specific artifact to see its lineage over time. Additionally, you can inspect all versions of an artifact with a given key by running:
+In the Prefect UI, you can view all of the latest versions of your artifacts and click into a specific artifact to see its lineage over time.
+Additionally, you can inspect all versions of an artifact with a given key by running:
 
-<div class="terminal">
 ```bash
-$ prefect artifact inspect <my-key>
+prefect artifact inspect my_key
 ```
-</div>
 
 or view all artifacts by running:
 
-<div class="terminal">
 ```bash
-$ prefect artifact ls
+prefect artifact ls
 ```
-</div>
 
 You can also use the [Prefect REST API](https://app.prefect.cloud/api/docs#tag/Artifacts/operation/read_artifacts_api_accounts__account_id__workspaces__workspace_id__artifacts_filter_post) to programmatically filter your results.
 
-### Deleting Artifacts
+### Deleting artifacts
 
 You can delete an artifact directly using the CLI to delete specific artifacts with a given key or id:
 
-<div class="terminal">
 ```bash
-$ prefect artifact delete <my-key>
+prefect artifact delete my_key
 ```
-</div>
 
-<div class="terminal">
 ```bash
-$ prefect artifact delete --id <my-id>
+prefect artifact delete --id my_id
 ```
-</div>
 
 Alternatively, you can delete artifacts using the [Prefect REST API](https://app.prefect.cloud/api/docs#tag/Artifacts/operation/delete_artifact_api_accounts__account_id__workspaces__workspace_id__artifacts__id__delete).
 
 ## Artifacts API
 
-Prefect provides the [Prefect REST API](https://app.prefect.cloud/api/docs#tag/Artifacts) to allow you to create, read, and delete artifacts programmatically. With the Artifacts API, you can automate the creation and management of artifacts as part of your workflow.
+Prefect provides the [Prefect REST API](https://app.prefect.cloud/api/docs#tag/Artifacts) to allow you to create, read, and delete artifacts programmatically.
+With the Artifacts API, you can automate the creation and management of artifacts as part of your workflow.
 
-For example, to read the 5 most recently created markdown, table, and link artifacts, you can do the following:
+For example, to read the five most recently created Markdown, table, and link artifacts, you can do the following:
```

Comment:
```suggestion
For example, to read the five most recently created Markdown, table, and link artifacts, you can run the following:
```

Reply:
nit: I'd personally prefer to keep the "5" rather than "five" as I think it makes it easier to digest quantitative information quickly in technical documentation.